guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,6 +12,22 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Leapp utility does not properly enable the {Project} module.
+To work around to enable the {Project} module for {ProjectServer}, run the following command:
```

Comment:
```suggestion
Run the following command to work around the issue.
```

Reply:
As a little detail, don't forget colon at the end.